blog/app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ArticlesController < ApplicationController
 
-  http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
+  #http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
```

Comment:
you should just delete this line. if you ever want to find it again you can use git history to.
[this files history](https://github.com/kahlya/ruby-try-three/commits/master/blog/app/controllers/articles_controller.rb)